logd/store/api.go

```diff
@@ -2,17 +2,21 @@ package store
 
 import (
 	"context"
-	"io"
 
 	"github.com/deref/exo/logd/api"
 )
 
 type Store interface {
+	// Returns the next log after the given argument.
```

Comment:
It would be helpful to define what "next" means in this context. My immediate question is whether this would iterate through all logs available (regardless of what log I passed in first), or would or follow some natural ordering, potentially terminating without iterating all logs if I did not start at the "beginning".

Reply:
Good point. I'll improve this comment.